mungegithub/features/server.go

```diff
@@ -65,6 +65,7 @@ func (s *ServerFeature) Name() string {
 
 // Initialize will initialize the feature.
 func (s *ServerFeature) Initialize(config *github.Config) error {
+	config.ServeDebugStats("/stats")
```

Comment:
Does this work? config.ServeDebugStats uses http.HandleFunc, but I think s.ConcurrentMux uses a non-singleton http handler?

Reply:
See the next line. `s.ConcurrentMux` wraps the `http.DefaultServeMux` which is the mux that `http.HandleFunc` will register to.